### DISCUSSION TITLE
Additional Config for installation destination.

### DISCUSSION THREAD

**Discussion by athenaabcxyz** [ChrisTitusTech/winutil#821] [Ideas]
Greetings, thanks for the awesome project. 
I do have an idea. Can you make an additional config that let the user choose where to install all the apps? I do not usually install my app or game in C driver (OS driver). Instead, I will prefer another location.

Thank you very much.

**Comment by MSSFree88** [comment]
I agree with this. I also usually don't install things on my OS drive and would like to put it on another drive.
